client/templates/hangout/hangout-item.js

```diff
@@ -47,17 +47,18 @@ Template.hangoutItem.helpers({
   },
   getDate: function(hangout) {
     var tz = TimezonePicker.detectedZone();
+    var startEndDiffDays = (hangout.end - hangout.start)/(1000*60*60*24)
+    var startTime = moment(hangout.start).tz(tz).format('ddd MMMM Do YYYY, h:mm a z')
+    // If endtime is in 24 hours (95% of 24 hours), then show "To Be Announced". Else show the end time
+    var endTime = startEndDiffDays < 0.95 ? moment(hangout.end).tz(tz).format('MMMM Do h:mm a z'): 'To Be Announced '
+    var joinedUsers = hangout.users.length + ' joined'
```

Comment:
Continues with the comment above:

The `joined` string should be moved as close as possible to the template where it is being used, in this case, in the template literal. 

Reason behind this: so when you take a peek at the template you would easily predict what the output would look like. Right now if you don't have any knowledge of how the code in this function looks like you could assume that the return would be something like this: `20:00 - 21:00 | 4`